£Lerror_reason¢DtypeEvalueYHTTPConnectionPool(host='www.ics.uci.eduauditortraining.csv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000004D87C50>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed',))Ihttp_code¢DtypeEvalueÛQdownload_complete¢DtypeEvalueı